Assets/BossRoom/Scripts/Server/ServerTestingHotkeys.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using MLAPI;
+using UnityEngine;
+
+namespace BossRoom.Server
+{
+    /// <summary>
+    /// Provides various special commands that the Host can use while developing and
+    /// debugging the game. Remove this component before releasing your game! (So that
+    /// regular players can't spawn extra bosses etc.)
+    /// </summary>
+    public class ServerTestingHotkeys : NetworkBehaviour
+    {
```

Comment:
What I did for previous projects was wrap everything inside a testing class with a #UNITY_DEBUG preprocessor directive. So essentially everything inside the class compiles only inside the editor and development builds, but not on release builds. The class itself always compiles, it's just empty on release. @pdeschain any objections?

Reply:
It doesn't seem like UNITY_DEBUG is pre-defined. Did you mean DEBUG? That seems to be auto-defined by Unity in the editor and in development builds, but not in release builds. (I've checked in that change for consideration.)

Reply:
Reading a bit more, seems the DEBUG symbol is coming from C# itself based on lower-level compilation directives. The best way I see to make our intent really explicit is to do `#if UNITY_EDITOR || DEVELOPMENT_BUILD`   so I switched to that.

Tested in editor, in develop builds, and in release builds, and works as intended.

Reply:
Ah yes, I did mean #DEBUG. But I like how explicit this is to a user.